CMakeLists.txt

```diff
@@ -72,7 +78,7 @@ option( BUILD_SHARED_LIBS "Build hipBLAS as a shared library" ON )
 
 
 # Hip headers required of all clients; clients use hip to allocate device memory
-find_package( hip REQUIRED CONFIG PATHS ${ROCM_PATH} /opt/rocm)
+find_package( HIP REQUIRED )
 
```

Comment:
Maybe just try
if (USE_CUDA) 
find_package HIP 
else()
find_package( hip REQUIRED CONFIG PATHS ${ROCM_PATH} /opt/rocm)

and then the romc builds are the same as hip provides the hip::host
One of the hip dependencies was the library I already removed (issues reported in 3.9), so ensuring all hip references are not in the if USE_CUDA may suffice.